packages/shopify-codemod/transforms/utils.js

```diff
@@ -1,4 +1,73 @@
 import j from 'jscodeshift';
+import chalk from 'chalk';
```

Comment:
This new code doesn't fit with what's already in the file (it's mostly AST element matchers/creators).  I'd prefer seeing it in a separate file (`code-utils.js`, or something).